xbridge_cli/bridge/build.py

```diff
@@ -52,26 +52,26 @@
     "--name",
     required=True,
     prompt=True,
-    help="The name of the bridge (used for differentiation purposes).",
```

Comment:
Why are these all changed to be action statements?

Reply:
Active voice is preferred for documentation--even with CLI help messages as far as I'm aware?